react/features/invite/components/InfoDialogButtonV2.web.js

```diff
@@ -0,0 +1,106 @@
+import InlineDialog from '@atlaskit/inline-dialog';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { translate } from '../../base/i18n';
+import { ToolbarButtonV2 } from '../../toolbox';
+
+import { updateDialInNumbers } from '../actions';
+
+import { InfoDialogButton, mapStateToProps } from './InfoDialogButton';
+import { InfoDialog } from './info-dialog';
+
+/**
+ * A React Component for displaying a button which opens a dialog with
+ * information about the conference and with ways to invite people.
+ *
+ * @extends InfoDialogButton
+ */
+class InfoDialogButtonV2 extends InfoDialogButton {
```

Comment:
Will this component be a substitution of InfoDialogButton? If yes, how do you plan to remove the old one if that's the superclass of this one? I guess you did this to reduce code duplication though, but maybe there is a better way (?)

Reply:
Yeah, I was thinking this morning I'd just take the original InfoDialogButton and split the render methods instead. The original plan was to swap the renders of InfoDialogbutton and V2 when it came time to remove the old web toolbar.